src/main/java/picard/sam/SamErrorMetric/ErrorType.java

```diff
@@ -13,10 +13,10 @@
  * This is used in {@link CollectSamErrorMetrics} to convert an input argument to a {@link BaseErrorAggregation}.
  */
 enum ErrorType implements CommandLineParser.ClpEnum {
-    ERROR(SimpleErrorCalculator::new, "Collects the average error at the bases provided."),
+    ERROR(SimpleErrorCalculator::new, "Collects the average (SNP) error at the bases provided."),
     OVERLAPPING_ERROR(OverlappingReadsErrorCalculator::new, "Only considers bases from the overlapping parts of reads from the same template. " +
-            "For those bases, it calculates the error that can be attributable to pre-sequencing, versus during-sequencing.");
-
+            "For those bases, it calculates the error that can be attributable to pre-sequencing, versus during-sequencing."),
+    INDEL_ERROR(IndelErrorCalculator::new, "Only collects insertion or deletion errors.");
```

Comment:
no need for "only"....
```suggestion
    INDEL_ERROR(IndelErrorCalculator::new, "Collects insertion and deletion errors at the bases provided.");
```